docs-kits/kits/Quality-Kit/Software Development View/page_software-development-view.md

```diff
@@ -11,7 +11,89 @@ sidebar_position: 4
 
 ## Quality Data Offers at EDC
 
-For the process described in the KIT Quality, data exchange between the participating partner companies is necessarily to be done for large vehicle and product populations. The data exchange therefore tshould be done as a file download via EDC according to the following specifications.
+When a Data Consumer calls the Catalog of a Data Provider, the Data Provider must signal in each Data Offer what exactly
+a Consumer could negotiate for. Data Offers in the Catalog are sorted by dcat:Datasets which are registered in the EDC-
+Management API as edc:Asset. Each Asset has private and public properties. The public properties are shown in the
+catalog and give hints to the Data Consumer what API and data they may expect. There are some properties that are
+mandatory for the entire Catena-X network and some that are mandatory only in specific Business Scenarios (like
+Quality).
+
+The dataAddress object's structure is determined by the dataplane implementation as it configures the details of the
+data transfer. They are not visible via the catalog.
+
+The following suggestion is a draft how Assets (and thus by proxy, dcat:Datasets) should be registered
+in the Quality Use-Case.
+
+```json
+{
+  "@context": {
+    "cx-taxo": "https://w3id.org/catenax/taxonomy#",
+    "cx-common": "https://w3id.org/catenax/ontology/common#",
+    "dct": "https://purl.org/dc/terms/",
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "@id": "someId",
+  "@type": "edc:Asset",
+  "edc:properties": {
```

Comment:
Should we also add the following property as the asset represents an aspect model - although it's not a submodel asset:

```"aas-semantics:semanticId": {"@id":  "urn:samm:io.catenax.vehicle.product_description:3.0.0#ProductDescription"}```

Reply:
This would be redundant to `dct:type`. That's not necessarily a problem but should be a conscious decision. Do you want that? If yes, I'll make sure to search for an appropriate standardized property-key that's not from the AAS-world.

Reply:
Not completely, right? dct:type does not contain any version information while the semnticId does. Of course, version could be specified in cx-common:version, but then we have to use the semantidId version there (3.0.0), nothing else. In the current example, it's different. Still, seems to be a clunky we to do it.

Reply:
- I have added the property `"dct:conformsTo": {"@id": "urn:samm:io.catenax.vehicle.product_description:3.0.0#ProductDescription"},` .
- We've also decided to have a `"dcat:keyword": "Quality"` property.

This leaves us with the question what to do with dct:type and cx-common:version. Options:
1.  `"dct:type": {"@id": "cx-taxo:QualityAsset"},` and increment cx-common:version in lockstep with CX-0123.
    i. remove the `dcat:keyword` property.
    ii. keep the redundant `dcat:keyword` property.
2.  `"dct:type": {"@id": "cx-taxo:<OneOfTheEightModels"},` and increment cx-common:version according to their semantic model's version.
    i. remove the `dct:conformsTo` property.
    ii. keep redundant `dcat:keyword` property.

I'd argue strongly for 1i.

Reply:
I prefer solution 1i as well. Redundant properties should not be kept. I am not sure what you mean with "increment cx-common:version in lockstep with CX-0123" - is CX-0123 a valid standard or do you mean that as an example? But what standard would you refer to then?

Reply:
CX-0123 is the standard that this structure will end up in - the standard "Quality Use-Case".
It will (afaik) be released with v1.0.0 in R24-05.